SslUsage/README.md

```diff
@@ -0,0 +1,480 @@
+- neither `key` not `cert` had an error loading
+
+##### Step 2 - connect the client w/ verification failing
+```
+F:\2019\qt5ex>SslUsage\SecureEchoService\Debug\SslUsage.SecureClient.exe
+
+20200701_215441.668  D  loadCert:38  error loading cert? false
+20200701_215441.672  I  loadCert:39  Loaded certificate:  QSslCertificate("1", "5c:57:ab:ab:07:39:b2:23:37:f7:55:36:a8:9c:2c:50:0a:7a:67:4e", "iRwlqgrODXY4RHdM2my3NA==", "CA", "client_01", QMap(), QDateTime(2020-07-01 19:26:41.000 UTC Qt::UTC), QDateTime(2020-07-31 19:26:41.000 UTC Qt::UTC))
+20200701_215441.684  D  loadKey:28  error loading key? false
+20200701_215441.692  I  loadKey:29  Loaded key:  QSslKey(PrivateKey, RSA, 4096)
+20200701_215441.699  I  main:58  connecting to "127.0.0.1" : 9876 ...
+20200701_215441.728  D  dumpSslErrors:86  SSL errors:
+20200701_215441.731  C  dumpSslErrors:89  "The host name did not match any of the valid hosts for this certificate"
+20200701_215441.736  D  dumpSslErrors:92  peer certificate chain:
+20200701_215441.741  D  dumpCert:70  ---
+20200701_215441.743  D  dumpCert:71  digest:  "55eecfb7a1130dbe30d900610e11b5ba"
+20200701_215441.747  D  dumpCert:72  serial:  "35633a35373a61623a61623a30373a33393a62323a32333a33373a66373a35353a33363a61383a39633a32633a35303a30613a37613a36373a3463"
+20200701_215441.756  D  dumpCert:73  ---subject
+20200701_215441.769  D  dumpCert:74  CN:  "server"
+20200701_215441.774  D  dumpCert:75  ORG: "github.com/ssproessig"
+20200701_215441.779  D  dumpCert:76  ---issuer
+20200701_215441.782  D  dumpCert:77  CN:  ("CA")
+20200701_215441.784  D  dumpCert:78  ORG: ("github.com/ssproessig")
+20200701_215441.787  D  dumpCert:79
+20200701_215441.793  D  dumpCert:70  ---
+20200701_215441.795  D  dumpCert:71  digest:  "29d034ad4cdee1a73f69c90fbfc81719"
+20200701_215441.799  D  dumpCert:72  serial:  "32643a35343a64613a61353a61643a61613a61623a63653a62353a32333a64313a61333a30373a66363a64313a39633a30313a35313a66663a3262"
+20200701_215441.807  D  dumpCert:73  ---subject
+20200701_215441.809  D  dumpCert:74  CN:  "CA"
+20200701_215441.812  D  dumpCert:75  ORG: "github.com/ssproessig"
+20200701_215441.817  D  dumpCert:76  ---issuer
+20200701_215441.819  D  dumpCert:77  CN:  ("CA")
+20200701_215441.829  D  dumpCert:78  ORG: ("github.com/ssproessig")
+20200701_215441.832  D  dumpCert:79
+20200701_215441.834  C  main:68  unable to connect securely
+```
+
+ensure neither `key` not `cert` had an error loading and note
+- handshake failed with `The host name did not match any of the valid hosts for this certificate`
+- the certificate chain is dumped, first the `server` certificated, then our `CA`
+
+Why did it fail? The server's log will not help us
+```
+20200701_215737.894  D  main::::operator():78  new client connected from "127.0.0.1" : 53010
+20200701_215738.013  D  main::::()::::operator():91  client from "127.0.0.1" : 53010 disconnected
+```
+obviously the client terminated the connection. Again, why Because per default we connect to
```

Comment:
This effect only really works when speaking. When written, it's kinda messy.

```suggestion
obviously the client terminated the connection. This happens, because per default we connect to
```